website/docs/guides/core-to-cloud-1.md

```diff
@@ -209,21 +225,21 @@ Explore these additional configurations to optimize your dbt Cloud orchestration
 
 ### CI/CD setup
 
-Building a custom solution to efficiently check code upon pull requests is complicated. With dbt Cloud, you can enable [continuous integration / continuous deployment (CI/CD)](/docs/deploy/continuous-integration) and configure dbt Cloud to run your dbt projects in a temporary schema when new commits are pushed to open pull requests. 
+Building a custom solution to efficiently check code upon pull requests is complicated. With dbt Cloud, you can enable [continuous integration / continuous deployment (CI/CD)](/docs/deploy/continuous-integration) and configure dbt Cloud to run your dbt projects in a temporary schema when new commits are pushed to open pull requests.
 
 This build-on-PR functionality is a great way to catch bugs before deploying to production, and an essential tool for data practitioners.
 
 1. Set up an integration with a native Git application (such as Azure DevOps, GitHub, GitLab) and a CI environment in dbt Cloud.
-2. Create [a CI/CD job](/docs/deploy/ci-jobs) to optimize workflows.
+2. Create [a CI/CD job](/docs/deploy/ci-jobs) to automate quality checks before code is deployed to production.
 3. Run your jobs in a production environment to fully implement CI/CD. Future pull requests will also leverage the last production runs to compare against.
 
-## Models configuration
+## Model development and discovery
```

Comment:
Is the "discovery" part the bit about Explorer? If so, we may want to replicate that language in the body below so the reader feels they've received the info from the title.

Reply:
this is more related to validating and discovering their models in cloud now. i might be wrong but i don't think the word discovery has anything to do with explorer - although it think explorer uses the discovery api maybe? idk i'd loop in @nghi-ly if we want more info!